packages/angular/src/utils/versions.ts

```diff
@@ -24,7 +24,6 @@ export const autoprefixerVersion = '^10.4.0';
 export const tsNodeVersion = '10.9.1';
 
 export const jestPresetAngularVersion = '~13.0.0';
-export const protractorVersion = '~7.0.0';
```

Comment:
Please check other package versions here that were only needed for Protractor and remove them as well.